<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUser</class>
 <widget class="QDialog" name="AddUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Contact</string>
  </property>
  <layout class="QHBoxLayout" name="_4">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QTextEdit" name="tb_info">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bitstream Vera Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To add a &lt;span style=&quot; font-weight:600;&quot;&gt;XMPP&lt;/span&gt; user, simply fill out the XMPP address (and optional nickname and group) at the bottom and press &lt;span style=&quot; font-style:italic;&quot;&gt;Add&lt;/span&gt;.&lt;br /&gt;&lt;br /&gt;To add a contact from a &lt;span style=&quot; font-weight:600;&quot;&gt;non-XMPP&lt;/span&gt; service, make sure you are registered with the service first (see Service Discovery from the main menu), and then select the service from the box at the top. Follow the instructions in the &lt;span style=&quot; font-style:italic;&quot;&gt;Service ID Translation&lt;/span&gt; box and press the &lt;span style=&quot; font-style:italic;&quot;&gt;Get XMPP Address&lt;/span&gt; button to generate an XMPP address for the contact.&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="w_serviceTranslation" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="hb_service">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lb_service">
            <property name="text">
             <string>Service:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_service">
            <item>
             <property name="text">
              <string>XMPP</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="Spacer1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="gb_trans">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Service ID Translation</string>
          </property>
          <layout class="QVBoxLayout" name="_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>11</number>
           </property>
           <item>
            <widget class="QLabel" name="lb_transDesc">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>No description</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="_3">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="le_transPrompt"/>
             </item>
             <item>
              <widget class="QPushButton" name="pb_transGet">
               <property name="text">
                <string>Get XMPP Address</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <layout class="QHBoxLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="le_jid"/>
         </item>
         <item>
          <widget class="IconToolButton" name="tb_vCard" native="true">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text" stdset="0">
            <string/>
           </property>
           <property name="psiIconName" stdset="0">
            <string>psi/vCard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_group">
         <property name="text">
          <string>Group:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="le_nick"/>
         </item>
         <item>
          <widget class="QToolButton" name="tb_resolveNick">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Resolve</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_nick">
         <property name="text">
          <string>Nickname (optional):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_jid">
         <property name="text">
          <string>XMPP Address:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cb_group">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_authreq">
       <property name="text">
        <string>Request authorization when adding</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Spacer3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>33</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_close">
       <property name="text">
        <string>Close window after adding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="Line2">
       <property name="frameShape">
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="BusyWidget" name="busy" native="true"/>
       </item>
       <item>
        <spacer name="Spacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>140</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="IconButton" name="pb_close" native="true">
         <property name="text" stdset="0">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="IconButton" name="pb_add" native="true">
         <property name="text" stdset="0">
          <string>&amp;Add</string>
         </property>
         <property name="shortcut" stdset="0">
          <string>Alt+A</string>
         </property>
         <property name="default" stdset="0">
          <bool>true</bool>
         </property>
         <property name="psiIconName" stdset="0">
          <string>psi/addContact</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BusyWidget</class>
   <extends></extends>
   <header>busywidget.h</header>
  </customwidget>
  <customwidget>
   <class>IconButton</class>
   <extends></extends>
   <header>iconbutton.h</header>
  </customwidget>
  <customwidget>
   <class>IconToolButton</class>
   <extends>QWidget</extends>
   <header>icontoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cb_service</tabstop>
  <tabstop>le_transPrompt</tabstop>
  <tabstop>pb_transGet</tabstop>
  <tabstop>le_jid</tabstop>
  <tabstop>le_nick</tabstop>
  <tabstop>cb_group</tabstop>
  <tabstop>ck_authreq</tabstop>
  <tabstop>ck_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
